doc/version_history.rst

```diff
@@ -2,6 +2,13 @@
 Version History
 ===============
 
+v2.0.0
```

Comment:
You sure these changes require a new major release? I don't know, they don't seem that big of a breaking change for a major review. 

I'd just do v1.4.0. 🤷‍♂️

Reply:
Fixed the version number in commit: [ef6deed](https://github.com/lsst-ts/ts_m2com/pull/50/commits/ef6deed7e99a1b45420bacee68d832ddd48c28b3). Thanks!